packages/text-area/theme/lumo/vaadin-text-area-styles.js

```diff
@@ -36,12 +36,26 @@ const textArea = css`
     border: none;
   }
 
-  :host(:hover:not([readonly]):not([focused]):not([invalid])) [part='input-field'] {
+  /* Allow pointer events when disabled in order to work around crash in FF
+     See https://bugzilla.mozilla.org/show_bug.cgi?id=1739079 */
+  :host([disabled]) {
+    pointer-events: initial;
+  }
+
+  /* Part of the workaround above: restore default cursor and prevent text 
+     selection, same as when setting pointer-events: none */
+  :host([disabled]) [part='input-field'],
+  :host([disabled]) ::slotted(textarea) {
+    cursor: default;
+    user-select: none;
+  }
```

Comment:
Using `pointer-events: none` also has the side-effect that text can not be selected, and that the cursor will not change to a text cursor when hovering the textarea.

Added this as an attempt to restore the existing behaviour.

Reply:
Hmm, preventing text selection doesn't seem to work in Chrome, Safari + iOS Safari. Maybe `user-select` is treated differently for input elements. Text selection also works for Chrome on master, so it's not consistent there either.

Reply:
Safari requires the `-webkit-` prefix for it https://caniuse.com/user-select-none

Reply:
Add a rule for Safari. Behavior should now match the one on master.